plugins/adobeair/plugin.js

```diff
@@ -125,14 +125,14 @@
 							// Do that before getDocumentHead because WebKit moves
 							// <link css> elements to the <head> at this point.
 							var div = new CKEDITOR.dom.element( 'div', doc );
-							div.setHtml( headHtml );
+							div.setHtml( CKEDITOR.tools.htmlSafeByReview(headHtml, 'Not too sure what is going on here') );
```

Comment:
Legacy conversion here since we're not sure how to refactor properly.

Reply:
Will be legacy converted